src/librustdoc/html/render/print_item.rs

```diff
@@ -1618,6 +1618,9 @@ fn item_variants(
         if let clean::VariantKind::Tuple(ref s) = variant_data.kind {
             write!(w, "({})", print_tuple_struct_fields(cx, s));
         }
+        if variant.attrs.other_attrs.iter().any(|a| a.has_name(kw::Default)) {
+            w.write_str(" <span class=\"stab default\">Default</span>");
```

Comment:
I forgot to mention that [people have shown an interest](https://rust-lang.zulipchat.com/#narrow/stream/393423-t-rustdoc.2Fmeetings/topic/meeting.2002-10-2023/near/394499427) in linking from the attribute to the default impl.
Not sure how easy would be to implement, you'd need to find said impl and link to it with [`format::href`](https://doc.rust-lang.org/nightly/nightly-rustc/rustdoc/html/format/fn.href.html). I think it can also be done as a follow-up.

Reply:
So as Manish said:
```
if you use #[default] it goes on the variant and otherwise it just shows up as a chip
```

I'll try to impl it.

Reply:
![image](https://github.com/rust-lang/rust/assets/45714/86ce6847-96ad-41f1-805c-699b801a8be0)

Here is the result.
The link goes to the trait impl below the screenshot.

@Manishearth since you initiated this, is it close enough to what you expected to see?

Reply:
I think we should also convert `(Non-exhaustive)` to a chip! But maybe for another ticket.
![image](https://github.com/rust-lang/rust/assets/45714/8e5ca8c4-0b24-400f-8490-067b4c629a1f)

Reply:
done (for Default)

Reply:
Right, okay. So I've re-read the relevant part of the meeting and I think I took Manish's comment out of context. They were talking about a hypothetical `#[doc(default)]` but not about `Default`'s `#[default]` :facepalm: Can sb. confirm that?

I don't think there was an interest in adding a such a hyperlink after all. I'm sorry that I made you to implement this logic.

Reply:
No problem at all :)

I also reread, I'm also confused for the final decision. Looks like people had different ideas and wishes, and didn't come to a single agreement :(

Reply:
Only the `impl Default for` indeed. There is not much point into linking to the attribute (as far as I can see at least).

Reply:
@GuillaumeGomez so the `Default` chip should be shown in front of `Variants`. It means I just need to remove the link to the Trait impl.

I did that because @Manishearth [asked for it](https://rust-lang.zulipchat.com/#narrow/stream/393423-t-rustdoc.2Fmeetings/topic/meeting.2002-10-2023/near/394499427).

Reply:
No no, the opposite: only generate a link to the implementation of the `Default` trait.

Reply:
OK, then what should be the text of the link? Is it the chip in front of  `Variants` text?
Because currently it _is_ a link to the `Default` impl.

Reply:
Then it's all good. I was replying to [this message](https://github.com/rust-lang/rust/pull/115575#discussion_r1369328054), sorry for the confusion.